README.md

```diff
@@ -6,16 +6,19 @@ A mighty, modern CSS linter and fixer that helps you avoid errors and enforce co
 
 ## Features
 
--   **Over one hundred and fifty rules:** Including those that:
-    -   **Catch errors**: e.g. invalid hex colors, duplicate selectors and overriding shorthand properties.
-    -   **Limit language features**: e.g. vendor prefixes, patterns for selectors and specific units, properties, functions and at-rules.
-    -   **Enforce stylistic conventions**: e.g. the whitespace around the colon in declarations and the number of empty lines between rules.
--   **Support for the latest CSS syntax:** Including custom properties, calc() and nesting.
--   **Understands *CSS-like* syntaxes:** The linter is powered by [PostCSS](https://github.com/postcss/postcss), so it understands any syntax that PostCSS can parse, including SCSS, [SugarSS](https://github.com/postcss/sugarss), and Less.
+-   **Over one hundred and fifty built-in rules.** Including ones that:
+    -   **Catch possible errors:** e.g. invalid standard CSS, duplicates and overrides.
```

Comment:
“invalid standard CSS” could be a little bit misleading. Because we're not validating whole CSS file, only parts of it. And we recommend [stylelint-csstree-validator](https://github.com/csstree/stylelint-validator) to validate CSS.

> e.g. invalid

“e.g., invalid”.

Should we use Oxford comma here?

Reply:
> “invalid standard CSS” could be a little bit misleading... And we recommend stylelint-csstree-validator to validate CSS.

Yes, I was torn about this. At some point, we'll likely hand over _all_ validation to csstree. However, at the moment we do have built-in rules that:
- validate at-rule, selector, etc. names.
- check for invalid `!important`s on keyframe declarations.

The "invalid standard CSS" is there as a nod to those. There's probably a better way to about it though. Any suggestions?

Umm, now that 

This reminds me that stylelint also surfaces invalid syntax errors (like unclosed braces) from PostCSS. I wonder if that should be mentioned in the features?

> “e.g., invalid”.

Do you mean?:

e.g., invalid standard CSS, duplicates, and overrides.

I'm happy to go with whichever you deem most correct :)

Reply:
> The "invalid standard CSS" is there as a nod to those. There's probably a better way to about it though. Any suggestions?

On the second thought, “invalid standard CSS” is quite correct. Maybe we should change it to “invalid standard CSS syntax” as it is a more correct description of built-in rules.

> e.g., invalid standard CSS, duplicates, and overrides.

Grammarly says this is correct :)

Reply:
> invalid standard CSS syntax as it is a more correct description of built-in rules.

Sure. I actually wrote that first and then shorten it. But yes, you're right.

> Grammarly says this is correct :)

The machines know best :)